launcher_scripts/conf/ptq/model/quantization.yaml

```diff
@@ -23,16 +23,21 @@ inference:
   max_context_length: 512 # max length of the context, input sequence will be truncated if it is longer than this
 
 trainer:
-  devices: ${..tensor_model_parallel_size}
-  num_nodes: ${..pipeline_model_parallel_size}
+  devices: ${model.tensor_model_parallel_size}
+  num_nodes: ${model.pipeline_model_parallel_size}
   accelerator: gpu
   logger: false # logger provided by exp_manager
   precision: bf16 # 16, 32, or bf16
   enable_checkpointing: false
 
+model:
+  tensor_model_parallel_size: 1
+  pipeline_model_parallel_size: 1
+  restore_from_path: llama2-7b-fp16.nemo # Nemo file path
+
 quantization:
-  quantize_bmm1: false
-  algorithm: fp8 # int8_sq, fp8, int8, int4_awq, null
+  decoder_type: ${export.decoder_type} # gptnext, gpt2, llama
+  algorithm: fp8 # null, int8, int8_sq, fp8, int4_awq, w4a8_awq, int4
```

Comment:
Could you also add `alpha` here? As per https://github.com/NVIDIA/NeMo/pull/9423 (I was thinking to make it optional but then decided for a fixed param in analogy to awq_block_size